src/server/server.rs

```diff
@@ -46,6 +46,8 @@ const MAX_GRPC_RECV_MSG_LEN: i32 = 10 * 1024 * 1024;
 pub const GRPC_THREAD_PREFIX: &str = "grpc-server";
 pub const STATS_THREAD_PREFIX: &str = "transport-stats";
 
+/// The TiKV server. It hosts various internal components, including gRPC,
```

Comment:
In Rustdoc, the first line is like a "Synopsis" line. On pages which show the short summary (eg [here](https://docs.rs/raft/0.4.0/raft/raw_node/index.html)) it will show the content of the first line.

So here it might be better to say `The TiKV Server.\nIt hosts...`